schema.py

```
@@ -0,0 +1,59 @@
+import json
+import time
+import uuid
+import ast
+import random
+import re
+import logging
+
+from console_exception import ConsoleException
+
+
+def generate(data, limit):
+    """
+    Generate JSON objects based on the schema.
+    :param data: JSON schema
+    :param limit: number of lines
+    :return: JSON objects
+    """
+    logging.info('Generate data based on the schema')
+    schema = json.loads(data)
+    lines = []
+    for i in range(limit):
+        line = {}
+        for key, value in schema.items():
+            if value:
+                params = value.split(":")
+                line[key] = _generate_type(params[0], params[1])
+        lines.append(line)
+    return lines
+
+
+def _generate_type(field_type, args):
+    switcher = {
+        "str": _generate_str,
+        "int": _generate_int,
+        "timestamp": _generate_time
+    }
+    func = switcher.get(field_type)
+    if not func:
+        raise ConsoleException("Type: {} not supported.".format(field_type))
+    return func(args)
+
+
+def _generate_str(args):
+    if args == "rand":
+        return str(uuid.uuid4())
+    elements = ast.literal_eval(args)
+    return random.choice(elements)
+
+
+def _generate_int(args):
```

I don't really like the idea of passing the `args` and `params` around and scattering the validation around many functions.
I think all the input handling logic should be concentrated in one place in the main module, and then you either fail early or pass properly named (and possibly type-annotated) arguments into the functions.